HOW_TO.md

```diff
@@ -60,8 +60,7 @@ to run it manually, you can do so as follows.  The test should take
 only a few seconds to run.
 
 ```
-$ PYTHONPATH=. python3 -m pytest -v
-tests/unittest/benchmark_runner/common/templates/
+$ PYTHONPATH=. python3 -m pytest -v tests/unittest/benchmark_runner/common/templates/
```

Comment:
This needs to be `tests/unittest/benchmark_runner/common/template_operations/`

Reply:
Fixed